entities/Users.ts

```diff
@@ -54,6 +54,20 @@ export class Users {
         const response = await this.api.get(`/users/${userID}/tracks`)
         return response as Promise<SoundcloudTrack[]>
     }
+    /**
+     * Gets all the tracks by the user using Soundcloud v2 API.
+     */
+    public tracksV2 = async (userResolvable: string | number) => {
+        const userID = await this.resolve.getV2(userResolvable)
+        const response = await this.api.getV2(`/users/${userID}/tracks`) as SoundcloudUserTrackCollectionV2
+        let nextHref = response.next_href;
+        while (nextHref) {
+            const nextPage = (await this.api.getURI(nextHref)).data as SoundcloudUserTrackCollectionV2;
```

Comment:
Rename `getURI` to `getURL` (it was never used so it won't break anything).

I don't like the parentheses so change the line to `const nextPage = await this.api.getURL(nextHref).then((r) => r.data)`